packages/react/scripts/generatePrimitiveCatalog.ts

```diff
@@ -66,6 +66,7 @@ const priorityProps = {
   ],
   Text: ['children', 'fontWeight', 'fontSize', 'color'],
   TextField: [...fieldProps, 'variation'],
+  TextAreaField: [...fieldProps, 'rows', 'maxLength', 'variation'],
```

Comment:
Do we need to include `'resize'` as well?

Reply:
Good catch, fixed